src/en/mangabtt/src/eu/kanade/tachiyomi/extension/en/mangabtt/MangaBTT.kt

```diff
@@ -0,0 +1,261 @@
+        searchMangaSelector()
+
+    override fun latestUpdatesFromElement(element: Element): SManga =
+        searchMangaFromElement(element)
+
+    override fun latestUpdatesNextPageSelector(): String =
+        searchMangaNextPageSelector()
+
+    // =============================== Search ===============================
+
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
+        val url = "$baseUrl/find-story".toHttpUrl().newBuilder().apply {
+            if (query.isNotBlank()) {
+                addQueryParameter("keyword", query)
+            } else {
+                val genre = filters.firstInstanceOrNull<GenreFilter>()?.selectedValue.orEmpty()
+                val status = filters.firstInstanceOrNull<StatusFilter>()?.selectedValue.orEmpty()
+                val sortBy = filters.firstInstanceOrNull<SortByFilter>()?.selectedValue.orEmpty()
+
+                addQueryParameter("status", status)
+                addQueryParameter("sort", sortBy)
+                if (genre.isNotBlank()) {
+                    addPathSegment(genre)
+                }
+            }
+
+            addQueryParameter("page", page.toString())
+        }
+
+        return GET(url.build(), headers)
+    }
+
+    override fun searchMangaSelector(): String = ".items > .row > .item"
+
+    override fun searchMangaFromElement(element: Element): SManga = SManga.create().apply {
+        thumbnail_url = element.selectFirst(".image img")?.imgAttr()
+        element.selectFirst("figcaption h3 a")!!.run {
+            title = text().trim()
+            setUrlWithoutDomain(attr("href"))
+        }
+    }
+
+    override fun searchMangaNextPageSelector(): String =
+        "ul.pagination > li.active + li:not(.disabled)"
+
+    // =============================== Filters ==============================
+
+    override fun getFilterList(): FilterList = FilterList(
+        Filter.Header("Ignored when using text search"),
+        Filter.Separator(),
+        GenreFilter(),
+        StatusFilter(),
+        SortByFilter(),
+    )
+
+    // =========================== Manga Details ============================
+
+    override fun mangaDetailsParse(document: Document): SManga = SManga.create().apply {
+        title = document.selectFirst("h1.title-detail")!!.text()
+        description = document.selectFirst(".detail-content p")?.text()
+            ?.substringAfter("comic site. The Summary is ")
+
+        document.selectFirst(".detail-info")?.run {
+            thumbnail_url = selectFirst("img")?.imgAttr()
+            status = parseStatus(selectFirst(".status p:not(.name)")?.text())
+            genre = select(".kind a").joinToString(", ") { it.text().trim() }
+            author = selectFirst(".author p:not(.name)")?.text()?.trim()?.takeUnless {
+                it.equals("updating", true)
+            }
+        }
+    }
+
+    private fun parseStatus(status: String?): Int = when {
```

Comment:
Consider making this a function under `Element`, as already done with `Element.imgAttr()` for consistency